application/controllers/handler.js

```diff
@@ -256,7 +263,7 @@ module.exports = {
       return res(boom.forbidden('You cannot change the user profile of another user'));
     }
 
-    console.log('updateUserProfile: use user', user);
+    console.log('updateUserProfile: use user', user.email, user.username);
```

Comment:
Is this enough information or would it be preferable to just strip the secret information by doing `delete user.XYZ;`